src/player.py

```diff
@@ -1,2 +1,15 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+from room import Room
+class Player:
+    def __init__(self, name, current_room, inventory):
+        self.name = name
+        self.current_room = current_room
+        self.inventory = inventory
```

Comment:
You could also write this as an empty list, but if it works, don't change it. It really depends on how you instantiate.